Console/retro/Console.cc

```diff
@@ -265,15 +265,79 @@ void Console::ScrollUp(short n)
     dirtyRect.bottom = dirtyRect.bottom > 0 ? dirtyRect.bottom - 1 : 0;
 }
 
+void Console::ProcessOSCseq(char c)
+{
+    switch(sequenceStep)
+    {
+    case 1:
+        if(c!='0')  // The only recognized sequence is OSC 0;
+        {
+            OSCseq=false;
+            sequenceStep=0;
+            return;
+        }
+        ++sequenceStep;
+        break;
+    case 2:
+        if(c!=';')  // The only recognized sequence is OSC 0;
+        {
+            OSCseq=false;
+            sequenceStep=0;
+            return;
+        }
+        ++sequenceStep;
+        windowName="";
+        break;
+    default:
+        if(c==BEL) // The BEL character ends the sequence.
+        {
+            setWindowName(windowName);
```

Comment:
use `setWindowName(std::move(windowName));` this way, the memory for the string won't be copied, but rather "moved" to `setWindowName` and then freed when it's done. As a nice side effect, we also save a few bytes for the window title (which would probably only be relevant on an 128K mac...)

Reply:
Here is an example of why I like to have my code reviewed. I just learnt something new, thank you very much!